### ISSUE TITLE
Goal tabs should reflect the Goal title

### ISSUE THREAD

**Issue by Norric1Admin** [ONSdigital/sdg-indicators#3090]
The ‘3 Good Health and Well-being’ page shares the same page title as the home page and does not uniquely reflect the contents of page. Page titles that do not uniquely describe or introduce the page content means that some users may not be able to identify the purpose or topic of the page content. “When I activated the third goal ‘good health and wellbeing’ link, I was confused by the fact that the URL changed, even though the page title did not.
`<title>U.K. Indicators For The Sustainable Development Goals</title>`

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility tickets](https://trello.com/c/E40VA79G/549-accessibility-tickets)

**Comment by LucyGwilliamAdmin** [comment]
## Description

## Screenshots
**Before**

**After**


## Successfully tested on...
- [] Windows: IE11
- [] Windows: Chrome
- [ ] Windows: Firefox
- [ ] Windows: Edge
- [ ] macOS: Chrome
- [ ] macOS: Firefox
- [ ] iOS: Safari 9.3
- [ ] iOS: Chrome
- [ ] Samsung: Chrome
- [ ] Samsung: Samsung Internet

## User feedback and problems encountered...

**Comment by Norric1Admin** [comment]
Indicator pages already have a specific tab - just remove the 'Indicator _._._' preface and leave the Goal details when browsing the Goal pages.

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Goal tabs should reflect the Goal title](https://trello.com/c/sOUXqD26/613-goal-tabs-should-reflect-the-goal-title)

**Comment by Norric1Admin** [comment]
@brockfanning Succeeded in changing the Goal icon with your help - no details though. Any thoughts?
Test branch: https://sdg.mango-solutions.com/a11y-goal-tab/3/
https://github.com/ONSdigital/sdg-indicators/commit/bee45253dca6c41d1d8d3c0fb9f13d1745ecc65b
Many thanks,
Chris
![image](https://user-images.githubusercontent.com/37659239/60269033-99655b00-98e5-11e9-8465-0357c8d96e7a.png)

**Comment by brockfanning** [comment]
I believe that means that `{{ goal_number }}` and `{{ goal.short }}` variables do not exist on goal pages. It looks like they only get set for indicator pages, in [this file](https://github.com/open-sdg/open-sdg/blob/master/_includes/indicator-variables.html). I guess that file does not get included on goal pages.

**Comment by Norric1Admin** [comment]
Unable to use open-sdg `remote_theme` branches, so overriding _layouts/goal.html locally for now.
https://github.com/ONSdigital/sdg-indicators/commit/5c4b440c4c4f9225872d5bd1804361ee5a4b56df (added line 2 - `{%- include indicator-variables.html -%}`)

**Comment by Norric1Admin** [comment]
This has resulted in the Goal tab reading `Goal 2 - ".
An improvement, but I'll see if I can get the short description too.

**Comment by Norric1Admin** [comment]
This is working as hoped! Ready to be tested @LucyGwilliamAdmin @SavvasStephanides

**Comment by Norric1Admin** [comment]
This update also caused the indicator tab and Goal icon to break. Fix: https://github.com/ONSdigital/sdg-indicators/commit/60dada91f34375bd04254774d21d9c4da01660de

**Comment by Norric1Admin** [comment]
This added the description but the icon is still broken and the tab number is missing. Reverting to the original tags as in https://github.com/ONSdigital/sdg-indicators/pull/3195 **fixes the indicator page but breaks the goal page**.

**Comment by SavvasStephanides** [comment]
@brockfanning What is the flag that indicates that a goal page is currently open? We have `page.indicator` to check when an indicator page is open, but not sure about a goal page. Thanks!

**Comment by dougmet** [comment]
`page.indicator` tells us that the page has that variable. You can look at the `_indicators` directory. You see that it has (e.g.)

```yaml
---
indicator: 1.2.1
layout: indicator
permalink: /1-2-1/
---

```

I wonder if a better way would be to switch (or case-when) on the `page.layout` variable. So something like:

```html
<title>
{% case page.layout %}
  {% when 'indicator' %}
    {{ page_title | escape }}
  {% when 'goal' %}
    Goal page.sdg_goal - page.short
  {% else %}
    {{ site.title | escape }}
{% endcase %}
</title>
```

Or we could make title a function somehow.

**Comment by dougmet** [comment]
To directly answer the question as to what's the equivalent to `page.indicator` for a goal. I'd say `page.sdg_goal`. If that exists it's a goal page because that's not stored in the indicator page header. Goal headers currently look like this:

```yaml
---
title: Ensure inclusive and equitable quality education and promote lifelong learning
  opportunities for all
short: Quality Education
permalink: /4/
redirect_from: /quality-education/
sdg_goal: '4'
color: '#c5192d'
layout: goal
---
```

**Comment by Norric1Admin** [comment]
@dougmet So could we have...?
if `page.sdg_goal` = https://github.com/open-sdg/open-sdg/commit/56f4509607d50e1db6d7323b0dcb3fe6f9445695 + https://github.com/ONSdigital/sdg-indicators/commit/f4859c6b44c28a8d94f379d74c81c3014d1ea78b + https://github.com/ONSdigital/sdg-indicators/commit/29b44260321e5d5efca046d1db233d8cda167fcb
elsif `page.indicator` = https://github.com/ONSdigital/sdg-indicators/pull/3195/files

**Comment by SavvasStephanides** [comment]
Thanks @dougmet, I think that answers my question. The way to do this (I think) is:

```
if page.indicator exists
    title = "Indicator <indicator number> - Goal <goal number>
else if page.sdg_goal exists
    title = "Goal <goal number> - <Goal description>"
else
    title = "<Site name>"
```

**Comment by SavvasStephanides** [comment]
`open-sdg` pull request: https://github.com/open-sdg/open-sdg/pull/229

**Comment by SavvasStephanides** [comment]
Pull request: https://github.com/open-sdg/open-sdg/pull/231

Test: https://sdg.mango-solutions.com/test-goal-tab

**Comment by LucyGwilliamAdmin** [comment]
This is fixed in Open SDG v0.8.0 so closing this.
